root/static/scripts/common/MB.js

```diff
@@ -50,7 +52,7 @@ MB.localStorage = function (name, value) {
 
         if (MB.hasLocalStorage) {
             try {
-                localStorage[name] = value;
+                window.localStorage.setItem(name, value);
```

Comment:
I've added a `node-storage-shim` dep to package.json to emulate local/sessionStorage under node, but it doesn't support this kind of property access, you have to use the class API.